maxim/logger/gemini/async_client.py

```
@@ -148,41 +180,78 @@ async def send_message_stream(
-                generation.result(response)
-            if self._is_local_trace:
-                self._logger.trace_end(self._trace_id)
-        except Exception as e:
-            logging.warning(
-                f"[MaximSDK][GeminiAsyncClient] Error in logging generation: {str(e)}"
-            )
-        # Returning response
-        return response
+
+        # Handle logging asynchronously without consuming the coroutine
+        async def handle_logging():
```

Ruff (0.11.9)

Missing return type annotation for private function `handle_logging`

Add return type annotation: `None`

(ANN202)